Makefile

```diff
@@ -131,3 +131,14 @@ update-chart: helm-lint
 	@helm repo index repo --url http://helm.k8s.magazineluiza.com
 	@aws s3 sync repo s3://helm.k8s.magazineluiza.com --delete
 	@rm -rf repo
+
+###Build Client in all Platforms
+PLATFORMS := windows linux darwin
+os = $(word 1, $@)
+
+.PHONY: $(PLATFORMS)
+$(PLATFORMS):
+	GOOS=$(os) go build -ldflags "-X $(BUILD_HOME)/pkg/version.Version=$(BUILD_VERSION)" -o teresa-$(os) $(BUILD_HOME)/cmd/client
+
+.PHONY: build-all-client
+build-all-client: windows linux darwin
```

Comment:
I GUESS these `PHONY` targets are not necessary, but Makefiles are black magic

Reply:
Yes, probably this work without the `PHONE`, I will try :)